src/Misc/fileTools.ts

```diff
@@ -87,7 +87,7 @@ export class FileTools {
      * Gets or sets a function used to pre-process url before using them to load assets
      */
     public static PreprocessUrl = (url: string) => {
-        return url;

```

Comment:
erroneous space removal?

Reply:
Is there a prettier config or auto-formatter that I am missing?

Reply:
updated!